web/src/compose_closed_ui.js

```diff
@@ -132,10 +132,6 @@ export function update_buttons_for_private() {
     // disable the [Message X] button when in a private narrow
     // if the user cannot dm the current recipient
     const disable_reply = true;
-    $("#compose_buttons .compose_reply_button").attr(
-        "data-tooltip-template-id",
-        "disable_reply_compose_reply_button_tooltip_template",
-    );
```

Comment:
Why is removing this block correct?

Reply:
We don't have a `disable_reply_compose_reply_button_tooltip_template` anymore, and it has since been replaced by `compose_reply_button_disabled_tooltip_template`.